classes/classes/Saves.as

```diff
@@ -1792,18 +1792,28 @@ public function loadGameObject(saveData:Object, slot:String = "VOID"):void
 		player.hipRating = saveFile.data.hipRating;
 		player.buttRating = saveFile.data.buttRating;
 		
+		if (player.lowerBody > 3 && player.lowerBody < 5) { //Bitch's way of saying 4. So we can delete LOWER_BODY_TYPE_CENTAUR
```

Comment:
Uhm ... that should read
```as3
		if (player.lowerBody == 4) {
```
then.
```
if (xyz > n - 1 && xyz < n + 1)
```
is kinda stupid, if xyz is an integer value.